draft-ietf-dnsop-serve-stale.md

```diff
@@ -297,25 +299,24 @@ document.
 
 The "client response timer" is another variable which deserves consideration. If this value is too short, there exists the risk that stale answers may be used even when the authoritative server is actually reachable but slow; this may result in suboptimal answers being returned. Conversely, waiting too long will negatively impact the user response. 
 
-Many resolvers implement prefetching of answers before the TTL has expired. If this has been attempted recently, and the authorative servers were determined to be not reachable at this time, the check can be skipped - the authors recommend that this value be the same as the check interval (see below). 
+Many resolvers implement prefetching of answers before the TTL has expired. If this has been attempted recently, and the authorative servers were determined to be not reachable at this time, the check can be skipped - the authors recommend that this value be the same as the check interval (see below).
 
-Another variable is how often to re-check if the authoritative server is available once it has initially been determined to be unreachable. If this variable is set too large, stale answers may continue to be returned even after the authoritative server is reachable. One of the motivations for serve-stale is to make the DNS more resilient to DOS attacks (and thereby make them less attractive as an attack vector). If the serve-stale technique becomes widely deployed, and this variable is too small, authoritative servers may be hit with a significant amount of traffic when they become reachable again. For this reason, the authors strongly recommend that this value not be set lower than 30 seconds. 
+Another variable is how often to re-check if the authoritative server is available once it has been determined to be unreachable. If this variable is set too large, stale answers may continue to be returned even after the authoritative server is reachable. If the serve-stale technique becomes widely deployed, and this variable is too small, authoritative servers may be hit with a significant amount of traffic when they become reachable again. For this reason, the authors strongly recommend that this value not be set lower than 30 seconds. 
```

Comment:
Ok. I still think it is worth having something saying that the very fact that serve-stale exists should decrease the occurrence of DoS attacks, but I'm not sure where to put it...

Reply:
I moved that sentence to the top of the draft. In this paragraph it felt out of place in terms of the level of detail.